.gitlab-ci.yml

```diff
@@ -151,8 +151,28 @@ deps_deb:
     - cd $CI_PROJECT_DIR/vendor && tar czf $CI_PROJECT_DIR/vendor.tar.gz .
   artifacts:
     paths:
-#      - $CI_PROJECT_DIR/vendor
-#      - $CI_PROJECT_DIR/venv
+      - $CI_PROJECT_DIR/version.txt
+      - $CI_PROJECT_DIR/go-pkg.tar.gz
+      - $CI_PROJECT_DIR/go-bin.tar.gz
+      - $CI_PROJECT_DIR/vendor.tar.gz
+    expire_in: 1 week
+
+deps_deb_py2:
```

Comment:
nice going. i discussed with zandre a bit, the giltab pipeline might be a tricky thing. What we could do is make use of parent-child pipeline (could also be a separate MR).

https://docs.gitlab.com/ee/ci/parent_child_pipelines.html

The setup could be:
- Have a separate .gitlab-ci-build.yaml that builds/tests py2/py3 based on an environment variable
- Have the main gitlab file run one of the two by default, both upon release and always have the option to manually kick of both on branches/master

Reply:
will tackle this in the next PR